src/Testcontainers/Containers/DockerContainer.cs

```diff
@@ -385,13 +397,10 @@ protected virtual async Task UnsafeStartAsync(CancellationToken ct = default)
       await this.UnsafeCreateAsync(ct)
         .ConfigureAwait(false);
 
-      await this.client.AttachAsync(this.container.ID, this.configuration.OutputConsumer, ct)
```

Comment:
This was such a useful feature. Don't understand why it got removed. In my case I use it to write to xunit's `ITestOutputHelper` and can see logs in IDE at runtime. Now the only option is to run a background task that will periodically call `GetLogsAsync`. For now, as a workaround to preserve the previous behavior, I override `UnsafeStartAsync` and call `AttachAsync` via reflection.